PR: https://github.com/odoo/odoo/pull/178200

From: f10bd8ddcf99c9247dca417fefbc260454d74ea1
From: Gorash
Date: 2024-10-15 09:33:02

Breaking data model changes scores: del:4 + add:0, change matches:
-    _inherit = 'pos.config'
-    _inherit = 'pos.order'
-    _inherit = 'pos.order.line'
-    _inherit = 'pos.session'

Total Changes: 18

[IMP] *: adapt model class names to correspond to model names (apply script)

Part-of: odoo/odoo#178200
Related: odoo/enterprise#69762
Related: odoo/documentation#11115
Related: odoo/upgrade#6577
Related: odoo/design-themes#988
Related: odoo/upgrade-util#143

================================= pseudo patch: =================================

--- a/addons/pos_event/models/event_event.py
+++ b/addons/pos_event/models/event_event.py
@@ -1,8 +1,7 @@
 from odoo import api, fields, models
 
 
-class Event(models.Model):
-    _name = 'event.event'
+class EventEvent(models.Model):
     _inherit = ['event.event', 'pos.load.mixin']
 
     image_1024 = fields.Image("PoS Image", max_width=1024, max_height=1024)

--- a/addons/pos_event/models/event_question.py
+++ b/addons/pos_event/models/event_question.py
@@ -4,7 +4,6 @@ from odoo import api, models
 
 
 class EventQuestion(models.Model):
-    _name = 'event.question'
     _inherit = ['event.question', 'pos.load.mixin']
 
     @api.model

--- a/addons/pos_event/models/event_question_answer.py
+++ b/addons/pos_event/models/event_question_answer.py
@@ -4,7 +4,6 @@ from odoo import api, models
 
 
 class EventQuestionAnswer(models.Model):
-    _name = 'event.question.answer'
     _inherit = ['event.question.answer', 'pos.load.mixin']
 
     @api.model

--- a/addons/pos_event/models/event_registration.py
+++ b/addons/pos_event/models/event_registration.py
@@ -3,7 +3,6 @@ from odoo import fields, models, api
 
 
 class EventRegistration(models.Model):
-    _name = 'event.registration'
     _inherit = ['event.registration', 'pos.load.mixin']
 
     pos_order_id = fields.Many2one(related='pos_order_line_id.order_id', string='PoS Order')

--- a/addons/pos_event/models/event_registration_answer.py
+++ b/addons/pos_event/models/event_registration_answer.py
@@ -4,7 +4,6 @@ from odoo import api, models
 
 
 class EventRegistrationAnswer(models.Model):
-    _name = 'event.registration.answer'
     _inherit = ['event.registration.answer', 'pos.load.mixin']
 
     @api.model

--- a/addons/pos_event/models/event_ticket.py
+++ b/addons/pos_event/models/event_ticket.py
@@ -2,8 +2,7 @@
 from odoo import models, api, fields
 
 
-class EventTicket(models.Model):
-    _name = 'event.event.ticket'
+class EventEventTicket(models.Model):
     _inherit = ['event.event.ticket', 'pos.load.mixin']
 
     @api.model

--- a/addons/pos_event/models/pos_config.py
+++ b/addons/pos_event/models/pos_config.py
@@ -3,7 +3,7 @@ from odoo import models
 
 
 class PosConfig(models.Model):
-    _inherit = 'pos.config'
+    _inherit = ['pos.config']
 
     def _update_events_seats(self, events):
         data = []

--- a/addons/pos_event/models/pos_order.py
+++ b/addons/pos_event/models/pos_order.py
@@ -3,7 +3,7 @@ from odoo import models, fields, api
 
 
 class PosOrder(models.Model):
-    _inherit = 'pos.order'
+    _inherit = ['pos.order']
 
     attendee_count = fields.Integer('Attendee Count', compute='_compute_attendee_count')
 

--- a/addons/pos_event/models/pos_order_line.py
+++ b/addons/pos_event/models/pos_order_line.py
@@ -3,7 +3,7 @@ from odoo import models, fields, api
 
 
 class PosOrderLine(models.Model):
-    _inherit = 'pos.order.line'
+    _inherit = ['pos.order.line']
 
     event_ticket_id = fields.Many2one('event.event.ticket', string='Event Ticket')
     event_registration_ids = fields.One2many('event.registration', 'pos_order_line_id', string='Event Registrations')

--- a/addons/pos_event/models/pos_session.py
+++ b/addons/pos_event/models/pos_session.py
@@ -3,7 +3,7 @@ from odoo import api, models
 
 
 class PosSession(models.Model):
-    _inherit = 'pos.session'
+    _inherit = ['pos.session']
 
     @api.model
     def _load_pos_data_models(self, config_id):
